### PR TITLE
Feat/louis/arb bot improvements

### PR DESCRIPTION
**Pull request by Louis-Amas**

### CODE REVIEW

File: packages/bot-arbitrage/test/integration/arbBot.integration.test.ts [hunk 1]
```diff
@@ -404,8 +627,151 @@ describe("ArbBot integration tests", () => {
...
+        }
+      );
+      const txActivate = await activateTokensWithMgv(
+        [market.base.address, market.quote.address],
+        mgvDeployer
+      );
+      await mgvTestUtil.waitForTransaction(txActivate!);
+      const makerAddress = await this.accounts.maker.address;
+      const makerBaseBeforeBalance = await market.base.balanceOf(makerAddress);
+      const makerQuoteBeforeBalance = await market.quote.balanceOf(
```
Comment (peterMangrove) [inline packages/bot-arbitrage/test/integration/arbBot.integration.test.ts:692]: These are never used

File: packages/bot-arbitrage/test/integration/arbBot.integration.test.ts [hunk 2]
```diff
@@ -404,8 +627,151 @@ describe("ArbBot integration tests", () => {
         holdingDaiBeforeBalance < holdingWethfterBalance,
         "Should have gained holding token"
       );
+      /* check if the offer has been partially filled */
+      assert.equal(makerBaseAfterBalance.toNumber() > 99, true);
+      assert.equal(makerQuoteAfterBalance.toNumber() < 1, true);
+    });
+
+    it(`should find arb and do arb, bid`, async function () {
```
Comment (peterMangrove) [inline packages/bot-arbitrage/test/integration/arbBot.integration.test.ts:635]: Same name as the first test. Does it test the same thing?

**Review by peterMangrove — CHANGES_REQUESTED** [review]

File: packages/bot-arbitrage/test/integration/arbBot.integration.test.ts (hunk 2 above)
Comment (Louis-Amas) [reply packages/bot-arbitrage/test/integration/arbBot.integration.test.ts]: No it does a partial fill I addressed the naming :)

**Review by peterMangrove — DISMISSED** [review]

File: packages/bot-arbitrage/test/integration/arbBot.integration.test.ts [hunk 3]
```diff
@@ -730,6 +730,194 @@ describe("ArbBot integration tests", () => {
...
+      assert.ok(
+        quoteBeforeBalance < quoteAfterBalance,
+        "Should have gained quote"
+      );
+
+      assert.equal(makerBaseAfterBalance.toNumber(), 99);
+      assert.equal(makerQuoteAfterBalance.toNumber(), 100001);
+    });
+
+    it(`should find arb and do arb, bid`, async function () {
```
Comment (peterMangrove) [inline packages/bot-arbitrage/test/integration/arbBot.integration.test.ts:828]: Should this be partial arb ?

**Review by peterMangrove — CHANGES_REQUESTED** [review]

**Review by peterMangrove — APPROVED** [review]

**Pull request merged**